src/strategy/strategy.ts

```diff
@@ -17,4 +17,6 @@ export class Strategy {
 export interface StrategyTransportInterface {
     name: string;
     parameters: any;
+    operator?: string;
```

Comment:
Could the operator be an `Enum` type instead? 

```typescript
enum StrategyGroupOperator {
    AND,
    OR
}
```

Reply:
Indeed!